add: youtube link na doc (#1010)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/upstash-geolocation.mdx

<LinkButton link="/en/documentation/products/guides/configure-a-domain/" label="go to configuring a domain guide" outlined /> 

> Discover how to enhance your web application's performance with our latest tutorial on 'Rate Limit with Penalty on Azion + Upstash DB.' Watch now on Azion's YouTube channel.

Reviewer: ```suggestion
Discover how to enhance your web application's performance with this tutorial on **Rate Limit with Penalty on Azion + Upstash DB**. Watch now on Azion's YouTube channel.
```
